.github/workflows/integration_tests.yaml

```diff
@@ -10,14 +10,41 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+
+      # To speed up the tests, we build the charm with Charmcraft and put the
+      # `.charm` file in the repository root directory. Integration tests will
+      # pick up this file and avoid building it per test case.
+      - name: Install Charmcraft
+        run: sudo snap install charmcraft --classic
+      - name: Build charm
```

Comment:
Can't we use the action?

Reply:
There's a build action in `operator-workflows` repo, [here](https://github.com/canonical/operator-workflows/blob/main/internal/build/action.yaml), but it's meant to be used as an internal action (from other workflows). Do you know any other action for building a charm?